spec/lib/tasks/review_profile_spec.rb

```diff
@@ -39,6 +43,33 @@
         invoke_task
       end
     end
+
+    context 'when the user does not exist' do
+      let(:user) { nil }
+      let(:uuid) { SecureRandom.uuid }
```

Comment:
Recommend fixed string, like the comment below.

Reply:
updated in 62727baab77a5b37097f31c8759b98271c5de206